bin/lib/fafScraper.js

```diff
@@ -17,6 +17,47 @@ const cache = LRU({
 
 const fafAPI = "http://www.findanyfilm.com"
 
+function getCinemaByID(id)
+{
+  const fafURL = `${fafAPI}/api/screenings/by_venue_id/venue_id/${id}`;
+  if(id === undefined || id === "" || id === "0")
+  {
+    debug("getCinemaData: Request was rejected",id);
+    return Promise.reject("A cinema ID was not passed to the function");
+  }
+
+  const cachedVersion = cache.get(`cinema:${id}`);
```

Comment:
Can we make the key for the cache a little less generic? Maybe `cinemaInfo`? I'd prefer that we make variables and keys as verbose as possible so that developers who are reading through it in the future don't have to use up to much headspace remembering what variables are being used where.